apps/condo/domains/ticket/utils/clientSchema/search.ts

```diff
@@ -144,6 +132,46 @@ export function searchOrganizationProperty (organizationId) {
     }
 }
 
+export function searchAllOrganizationProperties (initialProperties) {
+    return function (organizationId) {
+        if (!organizationId) return
+        return async function (client, searchText, query = {}, first = 10, skip = 0) {
+            const where = {
+                OR: [
+                    {
+                        AND: [
+                            {
+                                id_in: initialProperties,
+                                OR: [
+                                    { deletedAt: null },
```

Comment:
it's better not to use deletedAt on the client side

Reply:
We need to show tickets on deleted properties

Reply:
Why?
If we delete property, when we usually does not maintain anything related to it